j2k/testData/fileOrElement/methodCallExpression/stringMethods.kt

```diff
@@ -7,7 +7,7 @@ internal class A {
     @Throws(Exception::class)
     fun constructors() {
         String()
-        // TODO: new String("original");
+        "original"
```

Comment:
In `stringMethods` class constructor you used `String("original")`, here you put only its parameter. I am not sure if putting directly `"original"` here is a proper thing.